UnitsNet.Tests/CustomCode/HeatFluxTests.cs

```diff
@@ -0,0 +1,88 @@
+﻿//------------------------------------------------------------------------------
+// <auto-generated>
+//     This code was generated (once) by \generate-code.bat, but will not be
+//     regenerated when it already exists. The purpose of creating this file is to make 
+//     it easier to remember to implement all the unit conversion test cases.
+// 
+//     Whenever a new unit is added to this quantity and \generate-code.bat is run,
+//     the base test class will get a new abstract property and cause a compile error
+//     in this derived class, reminding the developer to implement the test case
+//     for the new unit.
+//
+//     See https://github.com/angularsen/UnitsNet/wiki/Adding-a-New-Unit for how to add or edit units.
+//
+//     Add CustomCode\Quantities\MyQuantity.extra.cs files to add code to generated quantities.
+//     Add Extensions\MyQuantityExtensions.cs to decorate quantities with new behavior.
+//     Add UnitDefinitions\MyQuantity.json and run GeneratUnits.bat to generate new units or quantities.
+//
+// </auto-generated>
+//------------------------------------------------------------------------------
+
+// Copyright (c) 2013 Andreas Gullberg Larsen (andreas.larsen84@gmail.com).
+// https://github.com/angularsen/UnitsNet
+// 
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+// 
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+// 
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+
+using System;
+using Xunit;
+
+namespace UnitsNet.Tests.CustomCode
+{
+    public class HeatFluxTests : HeatFluxTestsBase
+    {
+        protected override double BtuPerHourSquareFootInOneWattPerSquareMeter => 3.16998331e-1;
+        protected override double BtuPerMinuteSquareFootInOneWattPerSquareMeter => 5.28330551e-3;
+        protected override double BtuPerSecondSquareFootInOneWattPerSquareMeter => 8.80550918e-5;
+        protected override double BtuPerSecondSquareInchInOneWattPerSquareMeter => 6.11493693e-7;
```

Comment:
I can't seem to locate a source to verify this one, but my attempt on the calculator based on `BtuPerSecondSquareFootInOneWattPerSquareMeter ` gives me the same.

Reply:
I checked [Google](https://www.google.com/search?safe=off&ei=iufdWt2sD834_AaV9JPoCQ&q=1+BTU%2Fft%C2%B2%2Fs+to+W%2Fm%C2%B2&oq=1+BTU%2Fft%C2%B2%2Fs+to+W%2Fm%C2%B2&gs_l=psy-ab.3..35i39k1.28407.28748.0.28851.3.3.0.0.0.0.55.154.3.3.0....0...1.1.64.psy-ab..0.2.98...0i8i10i30k1.0.P-8Iy3oyk_A) and [WolframAlpha](http://www.wolframalpha.com/input/?i=1.000000000+BTU%2Fs%C2%B7ft%C2%B2+to+W%2Fm%C2%B2)